build.gradle

```
@@ -29,7 +29,7 @@ ext.slf4jVersion = "1.7.30"
 ext.apacheCommonsVersion = "3.8.1"
 
 allprojects {
-    group = 'org.ballerinalang'
+    group = 'io.ballerina'
```

This should be `io.ballerina.stdlib`, but we cann use the gradle variable.

```suggestion
    group = project.group
```